### ISSUE TITLE
Add external information on proba of presence absence per species

### ISSUE THREAD

**Issue by kunstler** [kunstler/FUNDIV_sAPROPOS#21]
Two publications provide information on the probability of presence/absence of the main European tree species based on much larger data set than the one of FUNDIV. It would be interesting to get this information for all plots. This would allow to discuss where the plots are in term of the distribution / niche.

The first source is from jrc:
http://forest.jrc.ec.europa.eu/european-atlas-of-forest-tree-species/atlas-data-and-metadata/

To generate link per species to download directly in R we can use https://w3id.org/mtv/FISE/map-data-RPP/v0-3-2/internet/Abies-alba
changing the species name.

The second one is from EFI:
http://www.efi.int/portal/virtual_library/information_services/mapping_services/tree_species_maps_for_european_forests/

**Comment by kunstler** [comment]
JRC provides two type of raster 

- Maximum Habitat Suitability which is climatic suitability
- Relative Probability of Presence which includes forest cover (coniferous, broadleaved) so this is both climatic suitability and observed actual forest cover.

Maximum Habitat Suitability would probably be better but is unfortunately available for only half of the species (missing species *Pinus pinaster, Picea abies, Pinus nigra, Quercus ilex, Quercus petraea, Quercus pyrenaica, Quercus suber, Pinus pinea*, and *Quercus faginea*).

We could instead fit our own SDM to the FUNDIV data and potentially add other sources of data such as:
-GBIF
-BIEN
-Mauri, A. et al. 2017. EU-Forest, a high-resolution tree occurrence dataset for Europe. - Scientific Data 4: 160123.

The simplest solution would be to use [BIOMOD2](https://cran.r-project.org/web/packages/biomod2/biomod2.pdf) with the following four methods: generalized linear models, generalized boosted regression models, random forest and maximum entropy modelling.

**Comment by kunstler** [comment]
I started an other repo to fit SDM model using biomod2 using large scale European tree species distribution data (larger than the one of FUNDIV but with only presence absence) (see https://www.nature.com/articles/sdata2016123) with the [CHELSA](http://chelsa-climate.org/) climatic data and some aridity index for [P - PET]( http://www.cgiar-csi.org/data/global-aridity-and-pet-database) and [soil water content](http://www.cgiar-csi.org/data/global-high-resolution-soil-water-balance). I started to use ensemble modelling for all our species. I will try to involve a PhD student that I'm co-supervising with Wilfried Thuiller and Wilfried to help with that (offering them to be involve on the paper) as they have more expertise on SDM than me.

For the repo see
https://gitlab.irstea.fr/georges.kunstler/EUForestSDM

The Mauri data:
![sdata2016123-f1](https://user-images.githubusercontent.com/4505188/37354896-9510a6be-26da-11e8-81d6-7738dd4ed5d0.jpg)

**Comment by kunstler** [comment]
Currently ensemble SDM with biomod2 for all species. Waiting for comments from Wilfried on the approach.

**Comment by kunstler** [comment]
@sophiathirza I have all my SDM models fitted with biomod2 but now I need to predict them on the FUNDIV plots. I was wondering if you have more accurate coordinates of the plots that the one I have in the data? I yes could you extract the climatic data I'm using for the SMD?

Here are the functions I use to download the data https://www.dropbox.com/s/jwi3gx4927u5a8m/Climate.R?dl=0

The data are for Chelsa: Bio01, Bio04, Bio10, Bio11, Bio15, Bio16
and from other sources: aridity, alpha, AETy, Ph

If you don't have more accurate coordinates I can do that myself.

**Comment by sophiathirza** [comment]
I can probably get them for the German inventory and I can ask the Swedish and Finnish data providers, they might give them to us for this study. But I don't think that they are available for the Spanish. I can ask Paloma.

**Comment by kunstler** [comment]
Which coordinates did you used for the extraction of the climatic data that we are using for the IPM? The one in the data I have or something else? By how much are the coordinates in the data I have blurred?I just want to be coherent between what we did for the IPM and the SDM.Thanks for your help.-------- Message original --------Objet : Re: [kunstler/FUNDIV_sAPROPOS] Add external information on proba of presence absence per species (#21)De : Sophia À : kunstler/FUNDIV_sAPROPOS Cc : Georges Kunstler ,Author I can probably get them for the German inventory and I can ask the Swedish and Finnish data providers, they might give them to us for this study. But I don't think that they are available for the Spanish. I can ask Paloma.

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/kunstler/FUNDIV_sAPROPOS","title":"kunstler/FUNDIV_sAPROPOS","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/kunstler/FUNDIV_sAPROPOS"}},"updates":{"snippets":[{"icon":"PERSON","message":"@sophiathirza in #21: I can probably get them for the German inventory and I can ask the Swedish and Finnish data providers, they might give them to us for this study. But I don't think that they are available for the Spanish. I can ask Paloma."}],"action":{"name":"View Issue","url":"https://github.com/kunstler/FUNDIV_sAPROPOS/issues/21#issuecomment-377680679"}}}
